lib/ifupdown.sh.in

```diff
@@ -18,9 +18,9 @@ if [ -z "$brctl" ] || [ ! -x "$brctl" ]; then
   echo "'brctl' binary does not exist or is not executable" >&2
   exit 2
 fi
-ifconfig="$(which ifconfig)"
-if [ -z "$ifconfig" ] || [ ! -x "$ifconfig" ]; then
-  echo "'ifconfig' binary does not exist or is not executable" >&2
+ip="$(which ip)"
```

Comment:
i was trying to think if there are cases where `ifconfig` would still be needed, but `ip` has been around for a while

ack